### PR TITLE
[plugin.video.streamz@matrix] 1.0.0+matrix.1

### PR DESCRIPTION
**Pull request by michaelarnauts**
### Add-on details:

- **General**
  - Add-on name: Streamz
  - Add-on ID: plugin.video.streamz
  - Version number: 1.0.0+matrix.1
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/add-ons/plugin.video.streamz
  
Streamz is a video-on-demand platform offering Flemish content from DPG Media, Telenet and VRT, but also including 'Home of HBO' content and blockbuster movies.

### Description of changes:

v1.0.0 (2020-11-03)
- Initial release

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by michaelarnauts** [comment]
This addon is based on `plugin.video.vtm.go` what is already in the repo (for Leia). The code is really similar since they use the same backend, but it's a different service.

File: plugin.video.streamz/codecov.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+coverage:
```
Comment (enen92) [inline plugin.video.streamz/codecov.yml]: please drop (add to gitattributes as export-ignore if using CI/CD)

**Review by enen92 — CHANGES_REQUESTED** [review]
Code looks okay and ready to merge. The coverage test file has to go though

**Comment by michaelarnauts** [comment]
> Code looks okay and ready to merge. The coverage test file has to go though

Ow. Sorry. I've missed this. Is it okay to add the CHANGELOG? There have been comments about that in the past, but then I've read somewhere that it's okay to include them.

**Comment by enen92** [comment]
Changelog is fine but please keep the news tag in add-on.xml for your current version changes

**Comment by michaelarnauts** [comment]
> Changelog is fine but please keep the news tag in add-on.xml for your current version changes

Okay. Since this is the first actual release to the public, everything is new, so I've just put `Initial release` there. Usually, I put the most important changes there (since the actual changelog can be a bit technical).

**Review by enen92 — APPROVED** [review]

**Comment by michaelarnauts** [comment]
Thanks @enen92!

**Pull request merged**